packages/vscode-extension/src/chat/commands/create/createCommandHandler.ts

```diff
@@ -48,7 +50,7 @@ export default async function createCommandHandler(
 
   if (matchedResult.length === 0) {
     response.markdown(
-      "Sorry, I can't help with that right now. Please try to describe your app scenario.\n"
+      "No matching templates or samples found. Try a different app description or explore other templates.\n"
```

Comment:
Do we need to localize this one?

Reply:
Since other response are all in English, localize this string only looks a little bit strange. So keep it as it.